tests/OK/inductive_type.lp

```diff
@@ -0,0 +1,11 @@
+inductive nat : TYPE ≔
+ | z    : nat
+ | succ : nat → nat
+
+inductive list ≔
+ | nil  : list
+ | cons : a → list → list
```

Comment:
You probably don't want this example to be accepted as it contains an undefined symbol `a`. Perhaps replace `a` by `nat` for the moment.